holopy/inference/scipyfit.py

```diff
@@ -72,6 +72,7 @@ def residual(rescaled_values):
                 parameters, rescaled_values)
             noise = model._find_noise(unscaled_values, data)
             residuals = model._residuals(unscaled_values, data, noise)
+            # FIXME why isn't this just `sqrt(model.lnprior)`?
             prior = np.sqrt(guess_prior - model.lnprior(unscaled_values))
```

Comment:
Hey @barkls -- do you know why this line is
```python
prior = np.sqrt(guess_prior - model.lnprior(unscaled_values))
```
rather than just
```python
prior = np.sqrt(model.lnprior(unscaled_values)
```
Unless I'm missing something, the current version will (i) push the prior to the guessed values, not to the prior center, and (ii) if `model.lnprior > guess_prior` then there will be `nan`s here. 

The same line is in nmpfit as well.